src/main/java/org/example/homework1/AccountService.java

```diff
@@ -0,0 +1,72 @@
+package org.example.homework1;
+import java.time.Month;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class AccountService {
+
+    public static List<Account> getBalanceOver500(List<Account> accountsData){
+        return accountsData.stream()
+                .filter(a -> a.getBalance() > 500.0)
+                .toList();
+    }
+    public static Set<String> getCountry(List<Account> accountsData){
+        return accountsData.stream()
+                .map(Account::getCountry)
+                .collect(Collectors.toSet());
+
+    }
+    public static Boolean isYoungerThan2000(List<Account> accountsData){
+        return accountsData.stream()
+                .anyMatch(a -> a.getBirthday().getYear() < 2000);
+    }
+    public static Double sumOfBalancesForMaleAccounts(List<Account> accountsData){
+        return accountsData.stream()
+                .filter(a -> "Male".equals(a.getGender()))
```

Comment:
a.getGender().equals("Male") - prefered to use. 
but better create constant variable for  any string you use in code(or ENUM if it's used more then in 1 place)